roles/fuse-layer/tasks/main.yml

```diff
@@ -34,20 +34,11 @@
     group: root
   become: true
 
```

Comment:
Add ```notify: restart_fuser-layer``` to make sure EGA client is restarted after a changed config.ini

Reply:
Done. See new commit